examples/integration/xcodeproj_targets.bzl

```diff
@@ -47,6 +47,7 @@ XCODEPROJ_TARGETS = [
     "//tvOSApp/Test/UnitTests:tvOSAppUnitTests",
     "//watchOSApp/Test/UITests:watchOSAppUITests",
     "//watchOSAppExtension/Test/UnitTests:watchOSAppExtensionUnitTests",
+    "//iOSApp/Test/SwiftUnitTests:iOSAppSwiftUnitTests",
```

Comment:
Can you do this as it's own PR to keep the changes down?

Reply:
Before merging I would like this change in another PR landed, so the fixture shows the `env` set.